HCSP.js

```diff
@@ -0,0 +1,231 @@
+{
+    "translatorID": "3e375db7-5f6e-4861-ab8b-3ddb0f4accc9",
+    "label": "HCSP",
+    "creator": "Joris Muller",
+    "target": "^https?//www\.hcsp\.fr/[Ee]xplore.cgi/",
```

Comment:
Double backslashes if you're escaping inside the string. I'd recommend using Scaffold, which takes care of this. Also, you don't need to worry about case, since we do case-insensitive matching